app/Http/Middleware/CheckIfAlreadyVoted.php

```diff
@@ -36,12 +36,9 @@ public function handle($request, Closure $next)
 
         $poll = Option::findOrFail($request->input('option.0'))->poll;
 
-        //if we already have this user's IP stored and linked to the poll they are trying to vote on
-        if ($poll->ip_checking == 1) {
```

Comment:
Was this conditional check removed intentionally? I'm asking only because I had assumed that all of these changes were programmatic, style-only changes and want to ensure that this isn't some kind of parser bug.

Reply:
This was intentional.